PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -191,9 +195,16 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
                     resolution = JME::JetResolution::get(setup, m_jets_algo_pt);
                     resolution_sf = JME::JetResolutionScaleFactor::get(setup, m_jets_algo);
                 }
-            }
 
-            const JetCollection& jets = *jets_collection;
+                if(m_useDeterministicSeed) {
+                    unsigned int runNum_uint = static_cast <unsigned int> (event.id().run());
+                    unsigned int lumiNum_uint = static_cast <unsigned int> (event.id().luminosityBlock());
+                    unsigned int evNum_uint = static_cast <unsigned int> (event.id().event());
+                    unsigned int jet0pt = uint32_t(jets.empty() ? 0 : jets[0].pt()/0.01);
+                    std::uint32_t seed = jet0pt + (lumiNum_uint<<10) + (runNum_uint<<20) + evNum_uint;
```

Comment:
this is out of sync with the /master version

Reply:
sorry, forgot to push...